util/run.bzl

```diff
@@ -230,7 +230,7 @@ def container_run_and_extract(name, image, commands, extract_file):
         image_tar = image_tar + ".tar",
         commands = commands,
         extract_file = extract_file,
-        output_file = extract_file.lstrip("/"),
+        output_file = name + extract_file,
```

Comment:
Do you still need the lstrip?

Reply:
I we leave the lstrip, then there will be no additional directory, and the name of the output file will be <name><extract_file>. If we remove the lstrip, then we add an additional name specific folder, and the extracted or the output file will be inside that folder, which I think is a bit clearer?

Reply:
Ah, I think I see. Thanks.